packages/common/src/filters/selectFilter.ts

```diff
@@ -229,8 +229,14 @@ export class SelectFilter implements Filter {
       values = Array.isArray(values) ? values : [values];
       this.$filterElm.multipleSelect('setSelects', values);
     }
-    this.getValues().length > 0 ? this.$filterElm.addClass('filled') : this.$filterElm.removeClass('filled');
-
+    if (this.getValues().length > 0 ) {
```

Comment:
ESLint throws an error on this line `error  There should be no space before this paren  space-in-parens` and is caused by the space after `> 0 `, just remove the space and it will then run the full CI/CD and unit tests

Reply:
Oops!!! again... ;-)
Now that I have enabled the option in the VSCode of "Format on save", it will be much better.
Thanks